docker-compose.yml

```diff
@@ -441,7 +441,7 @@ services:
       - GENNY_KOGITO_SERVICE_URL=http://alyson2.genny.life:${GADAQ_PORT}
       - GENNY_KOGITO_DATAINDEX_HTTP_URL=http://alyson.genny.life:${DATAINDEX_PORT}
       - GENNY_GADAQ_SERVICE_URL=http://alyson2.genny.life:${GADAQ_PORT}
-      - GENNY_KEYCLOAK_URL=https://keycloak-bali.gada.io
+      - GENNY_KEYCLOAK_URL=${GENNY_KEYCLOAK_URL}
```

Comment:
Since the variable name remains the same, the entire line could be removed.

Reply:
done